src/test/java/io/strimzi/kafka/bridge/utils/KafkaBridgeVersion.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.utils;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class KafkaBridgeVersion {
```

Comment:
I think that just an Utils class could be enough, not adding a specific class for every utility method we could need.